.hecat/url-check.yml

```diff
@@ -66,6 +66,7 @@ steps:
         - '^https://traq.io/' # hecat/python-requests bug? always returns 404 but the website works in a browser
         - '^https://kodi.tv/' # Always returns 403 but loads in browser
         - '^https://www.domoticz.com/' # Always returns 403 but loads in browser
-        - '^https://dev.webtrees.net/demo-stable/index.php?ctype=gedcom&ged=demo' # site takes a while to load,  times out with the default 10s timeout
-        - '^https://www.gnu.org/software/mailman/' # # site takes a while to load,  times out with the default 10s timeout
-        - '^https://goteleport.com/' # Always returns 403 but loads in browser
\ No newline at end of file
+        - '^https://dev.webtrees.net/demo-stable/index.php\?ctype=gedcom\&ged=demo' # site takes a while to load,  times out with the default 10s timeout
+        - '^https://www.gnu.org/software/mailman/' # site takes a while to load,  times out with the default 10s timeout
+        - '^https://goteleport.com/' # Always returns 403 but loads in browser
+        - '^https://elixi.re/' # site takes a while to load,  times out with the default 10s timeout
```

Comment:
elixi.re does not work at all for me (504 gateway timeout), might as well remove the homepage link (replace it with the source code link) for elixire in this PR